view/frontend/templates/js/replacejs.phtml

```diff
@@ -2,18 +2,11 @@
 /**
  * Copyright © 2013-2017 Bolt, Inc. All rights reserved.
  * See COPYING.txt for license details.
- */
-// @codingStandardsIgnoreFile
-?>
-<?php
-/**
+ *
  * Replace js template
  * Puts / replaces the checkout buttons according to defined selectors.
  * Maintains Bolt checkout functionality.
- *
- * @var $block \Bolt\Boltpay\Block\Js
  */
-
 if (!$this->isActive()) return;
 ?>
```

Comment:
Hmm, looks like there is no corresponding open `<?php`
Is it right?

Reply:
Yes there is, several lines ahead, on line1.
<?php
/**
 * Copyright © 2013-2017 Bolt, Inc. All rights reserved.
 * See COPYING.txt for license details.
 *
 * Replace js template
 * Puts / replaces the checkout buttons according to defined selectors.
 * Maintains Bolt checkout functionality.
 */
if (!$this->isActive()) return;
?>